### DISCUSSION TITLE
Skip Settings screen after installation

### DISCUSSION THREAD

**Discussion by abvgit** [Open-Shell/Open-Shell-Menu#760] [Q&A] [answered]
Im looking for working solution

Not work:
HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu
"MenuStyle_Default"="Win7"

Not work:
REG ADD "HKCU\Software\OpenShell\StartMenu" /v "ShowedStyle2" /t REG_DWORD /d "1" /f

**Comment by blackcrack** [comment]
could you maybe say which WinNT you have and which Number of the Version of it..
and which version from the OpenShell ?

**Reply by abvgit** [reply]
win10 2004 
OpenShell 4.4.160

and no such option in admx

**Reply by blackcrack** [reply]
well, have also no clue.. i have only this for the silence install (just for store)

"OpenShellSetup_x_x_xxx.exe" /qn ADDLOCAL=StartMenu

For Startmenu only:
OpenShellSetup_x_x_xxx.exe /qn ADDLOCAL=StartMenu

and with ClassicIE : 
OpenShellSetup_x_x_xxx.exe /qn ADDLOCAL=StartMenu,ClassicExplorer,ClassicIE,ShellUpdate

and check the /? with the installer !


by the way, there is a possible for save the configuration,.... if you save the config and let copy back  .. behind install..
maybe a try worth ..

**Comment by See182** [comment]
I've found out that the settings screen pops up, if a cetain registry key is missing. In my case it was the key under HCU\Software\OpenShell\StartMenu\ShowedStyle2 . Everytime I removed this key, the settings windows always came up.

I've fixed this via GPO and ADMX.

You can try these options in Computer Configurations - Policy - Administrative Templates - Open-Shell - Open-Shell Menu
Menu style - Enabled - Windows 7

If you want to fix this without using GPOs you could import the registry key.

Best regards

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
I already answered it in the Classic Shell forum: http://www.classicshell.net/forum/viewtopic.php?f=7&t=1847 8 years ago.
